lib/engine.js

```diff
@@ -385,7 +387,7 @@ module.exports = function (s, conf) {
         }
         if (is_reorder && s[signal + '_order']) {
           if (signal === 'buy') {
-            reorder_pct = n(size).multiply(s.buy_order.price).divide(s.balance.currency).multiply(100).add(fee)
+            reorder_pct = n(size).multiply(s.buy_order.price).divide(s.balance.currency).multiply(100).add(s.buy_order.fee)
```

Comment:
`reorder_pct` becomes `buy_pct` and it used for calculating `tradeable_balance` and `size`.
and `fee` is considering again when calculating `tradeable_balance` and `size`.
so, I thought no need to add `fee` to `reorder_pct`.

and `so.buy_pct` and `reorder_pct` are becomes to `buy_pct` and it used for sizing.
If `so.buy_pct` is not contains `fee` then `reorder_pct` should not contains `fee`.

If I miss understand could you explain it for me?

Reply:
fee during a reorder is whatever fee was will creating the order, so we resize the order properly.

tradeable balance simply is the, well, amount of balance we are actually able to trade.

As no exchange would want to allow us to trade past the point the won't be able to pay our fees (can't pay if none is left after filling the order) we deduct it from balance prior to filling the order BUT only account for it if balance + fee actually excites 100%